docs/src/install/installation.rst

```diff
@@ -5,7 +5,7 @@
 .. _installation:
 
 ********************************************************************
-Installation
+Tensile installation
 ********************************************************************
 
 Install ROCm
```

Comment:
There are two headings in a row. Either remove this heading. 
Perhaps better to put an introduction on this page that explains what the purpose is.

Reply:
Review for "we" in this document

Reply:
> There are two headings in a row. Either remove this heading. Perhaps better to put an introduction on this page that explains what the purpose is.

There is only one heading. The first one you see in red is a removed line.

Reply:
> Review for "we" in this document

I found "we" in this statement:

"For demonstration purposes, we use the sample tuning file available in Tensile/Configs/rocblas_sgemm_example.yaml"
Even though I somehow feel that this sounds appropriate, I have replaced it with the following (if we need to avoid "we" at all costs):

"For demonstration purposes, the sample tuning file available in Tensile/Configs/rocblas_sgemm_example.yaml is used"

PS: I had read in google style guide, that we can use "we" in such cases. Let me know if I should refrain from "we".

Reply:
No opinion on this.